Project.toml

```diff
@@ -0,0 +1,32 @@
+name = "Blink"
+uuid = "ad839575-38b3-5650-b840-f874b8c74a25"
+authors = []
+version = "0.10.2"
+
+[deps]
+Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
+BinDeps = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
+Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
+JSExpr = "97c1335a-c9c5-57fe-bc5d-ec35cebe8660"
+JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+Lazy = "50d2b5c4-7a5e-59d5-8109-a42b560f39c0"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
+MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
+Mustache = "ffc61752-8dc7-55ee-8c37-f3e9cdd09e70"
+Mux = "a975b10e-0019-58db-a62f-e48ff68538c9"
+Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+Sockets = "6462fe0b-24de-5631-8697-dd941f90decc"
+WebIO = "0f1e0344-ec1d-5b48-a673-e5cf874b6c29"
+WebSockets = "104b5d7c-a370-577a-8038-80a2059c5097"
+
+[compat]
+JSExpr = "≥ 0.3"
+Lazy = "≥ 0.11.3"
+WebIO = "≥ 0.3"
```

Comment:
Should probably just use `"0.3, 0.4, 0.5, 0.6, 0.7, 0.8"` here.

Reply:
Hm, right, just `0.8` would probably be better. I really don't like not upper bounding deps, and we shouldn't claim compatibility with a (as of yet non-existant) WebIO v0.9.

Reply:
Thanks, I've tightened all the versions to be caret specifiers. I think this sounds good.